app/models/bet.rb

```diff
@@ -2,4 +2,41 @@ class Bet < ApplicationRecord
   has_many :users, through: :user_bets
   has_many :user_bets
   has_many :options, through: :user_bets, source: :bet_option
+
+  def resolve
+    losers.each do |debtor|
+      debt = amount_bet(debtor)
+      win_contributions.each do |contribution|
+        debtor.debts.create(creditor: contribution[:creditor],
```

Comment:
Indentation matters for readability...

This is something I should make sure rubocop is catching but accepted indentations are: 

```ruby
# in this indentation the closing parens lines up with the starting line of the function giving a visual 
# indication of what is encapsulated in the method call
debtor.debts.create(
  creditor: contribution[:creditor],
  amount:   debt * contribution[:percentage]
)
```

```ruby
# in this indentation the parens of the method being called line up indicating which arguments are being passed to a 
# particular method 
debtor.debts.create(
                     creditor: contribution[:creditor],
                     amount:   debt * contribution[:percentage]
                   )
```